modules/machineset-creating.adoc

```diff
@@ -78,37 +101,106 @@ agl030519-vplxk-worker-us-east-1e   0         0                             55m
 ----
 
-.. Check values of a specific compute machine set:
+.. To view values of a specific compute machine set custom resource (CR), run the following command:
 +
 [source,terminal]
 ----
 $ oc get machineset <machineset_name> -n \
      openshift-machine-api -o yaml
 ----
 +
+--
 .Example output
 [source,yaml]
 ----
-...
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  labels:
+    machine.openshift.io/cluster-api-cluster: <infrastructure_id> <1>
+  name: <infrastructure_id>-<role> <2>
+  namespace: openshift-machine-api
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+    machine.openshift.io/cluster-api-cluster: <infrastructure_id>
+    machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<role>
 template:
-    metadata:
-      labels:
-        machine.openshift.io/cluster-api-cluster: agl030519-vplxk <1>
-        machine.openshift.io/cluster-api-machine-role: worker <2>
-        machine.openshift.io/cluster-api-machine-type: worker
-        machine.openshift.io/cluster-api-machineset: agl030519-vplxk-worker-us-east-1a
-----
-<1> The cluster ID.
+  labels:
+    machine.openshift.io/cluster-api-cluster: <infrastructure_id>
+    machine.openshift.io/cluster-api-machine-role: <role>
+    machine.openshift.io/cluster-api-machine-type: <role>
+    machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<role>
+  spec:
+    providerSpec: <3>
+      ...
+----
+<1> The cluster infrastructure ID.
 <2> A default node label.
++
+[NOTE]
+====
+For clusters that have user-provisioned infrastructure, a compute machine set can only create `worker` and `infra` type machines.
+====
+<3> The values in the `<providerSpec>` section of the compute machine set CR are platform-specific. For more information about `<providerSpec>` parameters in the CR, see the sample compute machine set CR configuration for your provider.
+--
+
+ifdef::vsphere[]
+.. If you are creating a compute machine set for a cluster that has user-provisioned infrastructure, note the following important values:
++
+.Example vSphere `providerSpec` values
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+...
+template:
+  ...
+  spec:
+    providerSpec:
+      value:
+      apiVersion: vsphereprovider.openshift.io/v1beta1
+      credentialsSecret:
+        name: vsphere-cloud-credentials <1>
+      diskGiB: 120
+      kind: VSphereMachineProviderSpec
+      memoryMiB: 8192
+      network:
+        devices:
+          - networkName: "<vm_network_name>"
+      numCPUs: 4
+      numCoresPerSocket: 1
+      template: <vm_template_name> <2>
+        userDataSecret:
+          name: worker-user-data <3>
+        workspace:
+          datacenter: <vcenter_datacenter_name>
+          datastore: <vcenter_datastore_name>
+          folder: <vcenter_vm_folder_path>
+          resourcepool: <vsphere_resource_pool>
+          server: <vcenter_server_address> <4>
```

Comment:
Some issues with indentation and apiVersion is not correct. Here is a machineset's providerSpec example on vsphere. 
```
      providerSpec:
        value:
          apiVersion: machine.openshift.io/v1beta1
          credentialsSecret:
            name: vsphere-cloud-credentials
          diskGiB: 120
          kind: VSphereMachineProviderSpec
          memoryMiB: 16384
          network:
            devices:
            - networkName: qe-segment
          numCPUs: 4
          numCoresPerSocket: 4
          snapshot: ""
          template: rhcos-412.86.202303211731-0-vmware.x86_64.ova
          userDataSecret:
            name: worker-user-data
          workspace:
            datacenter: SDDC-Datacenter
            datastore: WorkloadDatastore
            folder: /SDDC-Datacenter/vm/newugd-35600-dpm9s
            resourcePool: /SDDC-Datacenter/host/Cluster-1/Resources
            server: vcenter.sddc-44-236-21-251.vmwarevmc.com
```